bananas_api/helpers/enums.py

```diff
@@ -109,3 +109,10 @@ class TerrainType(Enum):
     FLAT = "flat"
     HILLY = "hilly"
     MOUNTAINOUS = "mountainous"
+
+
+class Climate(Enum):
+    TEMPERATE = "temperate"
+    ARCTIC = "arctic"
+    TROPICAL = "tropical"
```

Comment:
In-game it is called `sub tropical` (and `sub arctic`). Do we want to match that?

Reply:
Yes, the GUI should use the same terms. Pretty sure someone has a strong opinion about subarctic climate not being all ice with white bears.